pkg/network/encoding/format.go

```diff
@@ -47,6 +62,7 @@ func FormatConnection(conn network.ConnectionStats) *model.Connection {
 	c.DnsCountByRcode = conn.DNSCountByRcode
 	c.LastTcpEstablished = conn.LastTCPEstablished
 	c.LastTcpClosed = conn.LastTCPClosed
+	c.DnsStatsByDomain = m
```

Comment:
I don't think we should  include both DnsFailureLatencySum and DnsStatsByDomain in they have the same information. Can we include only one based on whether or not domain collection is enabled  (with the understanding that we will need some extra logic in the backend to handle agents where this field is set)

Reply:
never mind, I missed that this was being done in state.go